api-report/container-loader.api.md

```diff
@@ -67,12 +67,16 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     // (undocumented)
     get closed(): boolean;
     // (undocumented)
+    connect(): void;
```

Comment:
What do you think of doing `connectToDeltaStream` and `disconnectFromDeltaStream`?  The pro in my mind is it's more explicit which is generally good, the con is it loses symmetry with `connected` event.  In other words, the end state of calling `connect()` is more than just a live websocket, it's that the container gets caught up and is fully connected and fires the `connected` event.

So... I think I take it back. But it's out there in case someone else thinks it's a better idea :)